scripts/download_image.py

```diff
@@ -79,14 +90,20 @@ def main(target='arm', site='wiki', save_dir=None, dry_run=False):
         logging.info("Save directory is created: %s", save_dir)
         os.makedirs(save_dir)
 
+    num_lines = -1
+    with open(filename, encoding="utf-8") as f:
+        for line in f:
+            num_lines += 1
+
     with open(filename, encoding="utf-8") as stream:
-        for url in _readlines(stream):
-            save_file = os.path.join(save_dir, url.split(separator)[-1])
+        for line, url in enumerate(_readlines(stream)):
+            file = url.split(separator)[-1]
+            progress(line, num_lines, status=os.path.abspath(os.path.join(save_dir, file)))
+            save_file = os.path.join(save_dir, file)
```

Comment:
enumerate() return the first number zero, is it ok for line count?
If not then enumerate(..., start=1) option for the initial number.
num_lines = -1 can be zero start.

"file" variable name
Python 2.x had "file" type object which open() returns.
Python editor/IDE may show "file" as keyword
like this in github code highlight shown. I suggest change the variable name
"name" or "save_name" ... whatever (note: "filename" is already used outer scope)